mw-webapp/src/component/button/Button.module.scss

```diff
@@ -1,14 +1,21 @@
 @import "src/styles/variables";
 
 .button {
-  width: 100px;
-  height: 40px;
+  padding: 13px 17px;
+  border: none;
+  border-radius: $primaryBorderRadius;
+  background-color: $inputInitial;
+  color: $whiteText;
```

Comment:
let's for color use naming: $primary, secondary, etc

Reply:
Now I don't what color will be the primary

Reply:
it doesn't matter, in the future we can easily rewrite it

Reply:
Done